src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function AuthorSearchInput({ text, onChange, classes }: any) {
```

Comment:
If we're accepting props here, we should have an interface to type check everything. Check out the other PRs on how to do this.

It should also not return `any` as that is pretty hacky. Please take a look at documentation or examples on what a functional component in react for typesript.